ts/Stock/Indicators/MACD/MACDIndicator.ts

```diff
@@ -479,18 +485,13 @@ class MACDIndicator extends SMAIndicator {
 
 interface MACDIndicator {
     crispCol: typeof ColumnSeries.prototype['crispCol'];
-    currentLineZone: (string|null);
     getColumnMetrics: typeof ColumnSeries.prototype['getColumnMetrics'];
-    graphmacd: (Highcharts.SVGElement|undefined);
-    graphsignal: (Highcharts.SVGElement|undefined);
-    macdZones: MACDZonesOptions;
     nameComponents: Array<string>;
     parallelArrays: Array<string>;
     pointArrayMap: Array<string>;
```

Comment:
`extend()` includes params, so you do not need them in the interface. TD:LD: `parallelArrays, pointValKey` etc should be removed here.

Reply:
Please do not remove them, if they were defined on the class in the internal namespace. As long as they are in the extend call, they should also be in the interface.

Reply:
I was pretty sure that its enough in the extend only, my mistake.

Reply:
Unfortunately the extend call is not type safe.